xcode/Subconscious/Shared/Components/Detail/MemoViewerDetailMetaSheetView.swift

```diff
@@ -0,0 +1,144 @@
+//
+//  MemoViewerDetailMetaSheetView.swift
+//  Subconscious (iOS)
+//
+//  Created by Gordon Brander on 4/6/23.
+//
+
+import SwiftUI
+import Combine
+import os
+import ObservableStore
+
+struct MemoViewerDetailMetaSheetView: View {
+    @Environment(\.dismiss) private var dismiss
+    var state: MemoViewerDetailMetaSheetModel
+    var send: (MemoViewerDetailMetaSheetAction) -> Void
+
+    var body: some View {
+        VStack(spacing: 0) {
+            HStack {
+                VStack(alignment: .leading, spacing: AppTheme.unit2) {
+                    HStack {
+                        if let slashlink = state.address?.toSlashlink() {
+                            SlashlinkBylineView(slashlink: slashlink).theme(
+                                petname: Color.primary,
+                                slug: Color.secondary
+                            )
+                        } else {
+                            Text("Draft")
+                                .foregroundColor(.secondary)
+                        }
+                    }
+                    .font(.callout)
+                }
+                Spacer()
+                CloseButtonView(action: { dismiss() })
+            }
+            .padding()
+            Divider()
+            ScrollView {
+                VStack(spacing: AppTheme.unit4) {
+                    MetaTableView {
+                        Button(
+                            action: {
+                                send(.copyLink)
+                            }
+                        ) {
+                            Label(
+                                "Copy link",
+                                systemImage: "doc.on.doc"
+                            )
+                        }
+                        .disabled(state.address == nil)
+                        .buttonStyle(RowButtonStyle())
+                    }
```

Comment:
To me, it seems cleaner to use `ShareLink` here instead of copying to the clipboard. I think it's more idiomatic in iOS in general to go via the share sheet since it offers copy to pasteboard within it anyway & it's more directly useful for sending a link in an iMessage etc. 

This apple link looked promising for guidance but I can't load it right now because of an SSL error(!!): https://developer.apple.com/design/human-interface-guidelines/patterns/collaboration-and-sharing/

We could expose a computed `shareableLink` property from the model to keep the logic from the `.copyLink` handler in the model, then use the `MetaTableItemShareLinkView` I implemented recently like so:

```swift
MetaTableItemShareLinkView(label: "Share Link", item: state.shareableLink)
```

Happy to merge this PR as-is, but I would vote to prefer `ShareLink` over manual pasteboard interaction as the default choice.